scripts/README.md

```diff
@@ -0,0 +1,24 @@
+# Utility Script
+
+This directory contains helper scripts for Charmed Kubeflow, during CI and not only.
+
+## Gather images used by a bundle
+
+You can get a list of all the OCI images used by the bundle by running the following command:
+```bash
+python3 scripts/get-all-images.py \
+    --append-images tests/airgapped/ckf-1.8-testing-images.txt \
+    releases/1.8/stable/kubeflow/bundle.yaml \
+    > images-all.txt
+```
```

Comment:
first you need to pip install `scripts/requirements.txt`

Reply:
good catch!
7d3358e

Reply:
(the commit is shown as not existing in the repository since I had to force-push to rebase on top of latest `main`)